pkg/security/probe/model_windows.go

```diff
@@ -21,6 +21,11 @@ func NewWindowsModel(_ *WindowsProbe) *model.Model {
 			// TODO(safchain) remove this check when multiple model per platform will be supported in the SECL package
 			if !strings.HasPrefix(field, "exec.") &&
 				!strings.HasPrefix(field, "exit.") &&
+				!strings.HasPrefix(field, "create_file.") &&
```

Comment:
I would use a 'file' event type with "file.create" ? so that later if we handle delete event it would be 'file.delete', etc.